packages/python-pygobject/python-pygobject.spec

```diff
@@ -6,7 +6,7 @@
 %global srcname pygobject
 
 Name:           %{?scl_prefix}python-%{srcname}
-Version:        3.42.0
+Version:        3.40.1
```

Comment:
i don't think we need to bother using epoch to try to make 3.40.1 newer than 3.42.0 as no one should have been able actually install this with the ostree plugin (and its the only thing that should require it)

Reply:
After we merge this, I'll need to untag the 3.42.0 package and sign 3.41.0

Reply:
https://github.com/theforeman/pulpcore-packaging/pull/371